libnmstate/schemas/operational-state.yaml

```diff
@@ -473,6 +474,20 @@ definitions:
                     type: boolean
                   state:
                     type: string
+  interface-team:
+    rw:
+      properties:
+        type:
+          type: string
+          enum:
+          - team
+        team:
+          type: object
+          properties:
+            slaves:
+              type: array
+              items:
+                type: string
```

Comment:
I think it needs to be an object and not a string.
Team ports have multiple options base on `nm-settings` man page.